cmd/image-processing/main.go

```diff
@@ -171,6 +178,38 @@ func runImageProcessing(ctx context.Context) error {
 		}
 	}
 
+	// check for image vulnerabilities if vulnerability scanning is enabled.
+	var vulns []shipwrightv1alpha1.Vulnerability
+	if flagValues.vulnerabilitySettings.Enabled && flagValues.push != "" {
+		imagePath := flagValues.push
+		if img != nil {
+			entries, err := os.ReadDir(flagValues.push)
+			if err != nil {
+				return err
+			}
+			imagePath = imagePath + "/" + entries[0].Name()
```

Comment:
Shouldn't that be done with `filepath.Join()`?

Reply:
Done